lib/events/new-editorial-comment.php

```diff
@@ -175,6 +175,28 @@ function comments_query( WP_Comment_Query $query ) {
 
 add_action( 'pre_get_comments', __NAMESPACE__ . '\comments_query' );
 
+/**
+ * Filter the comment count to exclude workflow comments.
+ *
+ * @param int $count
+ * @param int $post_id
```

Comment:
Missing parameter descriptions here
```suggestion
 * @param int $count The current number of comments found.
 * @param int $post_id The post ID to return the number of comments for.
```